app/views/content/life-as-a-teacher/explore-subjects/history/_article.html.erb

```diff
@@ -0,0 +1,78 @@
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching history opens the door for your pupils to pursue exciting careers in fields such as research, curation, and law.</p>
+      <p>Your pupils will learn more from you than you might think. As their history teacher, you'll develop your pupils’ knowledge and skills to investigate the past by interrogating primary and secondary sources.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_history_jack' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-history-programmes-of-study/national-curriculum-in-england-history-programmes-of-study">national curriculum for history</a>.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), themes you may study include:</p>
+    <ul>
+      <li>the development of church, state and society</li>
+      <li>ideas, political power, industry and empire: Britain, 1745-1901</li>
+      <li>challenges for Britain, Europe and the wider world from 1901 to the present day</li>
+      <li>a local history study</li>
+      <li>a study of a significant society or issue in world history and its interconnections with other world developments</li>
+    </ul>
+    <p>Knowledge and skills you'll cover when you teach 14 to 16 year olds (key stage 4) include :</p>
+    <ul>
+      <li>developing and extending knowledge of key historical events</li>
+      <li>developing questioning about the past through the use of sources</li>
+      <li>developing an awareness of why people, events and developments have been accorded historical significance</li>
+      <li>creating structured accounts using evidence</li>
+    </ul>
+    <p>Explore what teaching key stage 3 and 4 history lesson would be like by visiting:</p>
+    <ul>
+     <li><a href="https://www.thenational.academy/teachers/curriculum/history-secondary-aqa/units">AQA exam board lessons on the Oak Academy website</a></li>
+     <li><a href="https://www.thenational.academy/teachers/curriculum/history-secondary-edexcel/units">Edexcel exam board lessons on the Oak Academy website</a></li>
+    </ul>
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop further knowledge and skills in:</p>
+    <ul>
+      <li>being effective and independent students and as critical and reflective thinkers</li>
+      <li>acquiring an understanding of the nature of historical study</li>
+      <li>developing the use and understanding of historical terms, concepts and skills</li>
+      <li>making links and drawing comparisons within and/or across different periods and aspects of the past</li>
+      <li>organising and communicating historical knowledge and understanding in different ways and reaching substantiated judgements</li>
```

Comment:
is it worth simplifying `substantiated judgements`? eg conclusions based on evidence?

but I'll leave it to you two to decide!